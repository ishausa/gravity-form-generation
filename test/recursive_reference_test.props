event_title = This references another $place_holder
place_holder = This references the $date
date = today
