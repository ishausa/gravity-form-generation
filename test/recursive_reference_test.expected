Title = This references another This references the today

